### ISSUE TITLE
Cannot show image after image picking

### ISSUE THREAD

**Issue by zjxeditor** [luberda-molinet/FFImageLoading#295]
Hi, I am picking an local image from the gallery and then show it through CachedImage with Xamarin Forms. Here is the code:
````
await CrossMedia.Current.Initialize();
if (!CrossMedia.Current.IsPickPhotoSupported)
{
  return;
}
var image = await CrossMedia.Current.PickPhotoAsync();
if (image != null)
{
  //await Task.Delay(100);
  Image.Source = image.Path;
}
````

Everything works fine first. However, When I change the ImageSource, nothing shows up. If I use the Original Image control or add code `await Task.Delay(100);` before `Image.Source = image.Path;`, then the problem will be solved. 
So what's wrong when I use the CachedImage? It is very strange that `await Task.Delay(100);`  will make it work fine.

**Comment by daniel-luberda** [comment]
Hi @zjxeditor,

- Is it on Android / iOS / Windows?
- Can you make a simple reproduction so I could debug it?

**Comment by zjxeditor** [comment]
It is on Android. I am testing on a Nexus 5.

**Comment by daniel-luberda** [comment]
Do you experience it with newest stable?

**Comment by daniel-luberda** [comment]
@zjxeditor Do you experience it with newest stable?

**Comment by zjxeditor** [comment]
Yes, I will write a simple demo and send it to you.

**Comment by daniel-luberda** [comment]
Should be fixed in `2.1.9-pre-168-core-r`. Please reopen it if doesn't work for you.
